workspaces/scorecard/plugins/scorecard-backend-module-jira/src/constants/jiraOpenIssues.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Jira open issues thresholds configuration path
+ * @public
+ */
+export const THRESHOLDS_CONFIG_PATH =
+  'scorecard.plugins.jira.open_issues.thresholds' as const;
+
+/**
+ * Jira integration configuration path
+ * @public
+ */
+export const JIRA_CONFIG_PATH = 'jira' as const;
+
+/**
+ * Jira open issues options configuration path
+ * @public
+ */
+export const JIRA_OPTIONS_PATH =
+  'scorecard.plugins.jira.open_issues.options' as const;
+
+export const API_VERSION_DEFAULT = 'latest' as const;
+
+export const JIRA_MANDATORY_FILTER =
+  'type = Bug AND resolution = Unresolved' as const;
+
+export const CONFIG_BASE_URL = 'baseUrl' as const;
+
+export const CONFIG_TOKEN = 'token' as const;
+
+export const CONFIG_PRODUCT = 'product' as const;
+
+export const CONFIG_API_VERSION = 'apiVersion' as const;
+
+export const CONFIG_MANDATORY_FILTER = 'mandatoryFilter' as const;
+
+export const CONFIG_CUSTOM_FILTER = 'customFilter' as const;
```

Comment:
As said above and in our short meeting today. I would recommend to reuse the types of the Config (config.d.ts) and try to drop these keys.